src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let finalCost = 0;
+
+  for (let day = 0; day < days; day++) {
+    finalCost += 40;
```

Comment:
CL-1 [VARIABLES] - use variables for the main values so that you can reuse them and give them descriptive names.
how to understand what is 40?